common/components/controllers/MainController.jsx

```diff
@@ -34,7 +34,7 @@ class MainController extends React.Component<{||}> {
 
   render(): React$Node {
     return [
-      <MainHeader key='main_header'/>,
+      // <MainHeader key='main_header'/>,
```

Comment:
Can remove this to clean up the render function if we don't need it - I know you mentioned merging MainHeader and SubHeader in the PR. I'll check with Marlon but I don't see a problem having a single header component considering how simple our header is.